ci/container-images/permaplant-rust/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM rust:1.67.1-slim
+
+RUN apt update && apt install -y libpq-dev
+RUN rustup component add clippy
+RUN rustup component add rustfmt
```

Comment:
You can do the same as `rustup component add clippy rustfmt`.

Reply:
GREAT! That removes one layer :)

```suggestion
RUN rustup component add clippy rustfmt
```